django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
Of course, the above doesn't take into account the availability of the hasher, since PBKDF2 is available (via efficient C) in the stdlib in 2.7.9+ and 3.something+ and can be done reasonably in pure Python before that whereas scrypt, bcrypt, and argon2 will all require an external library (generally C based).